docs/02-for-app-authors/03-appdata-guidelines/01-quality-guidelines.md

```diff
@@ -0,0 +1,69 @@
+# Quality guidelines
+
+For the quick guidelines for the actual content (descriptions, screenshots), see the Appstream [docs](https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html).
+
+The following guidelines are optional, but recommended. We will likely promote apps that follow them in certain places.
+
+## General
+
+### No trademark violations
+
+Your app must not violate any trademarks. This includes the name, icon, and summary.
```

Comment:
```suggestion
Your app must not violate any trademarks. This includes the name, icon, and summary. For example, if your app is a Telegram client or web wrapper, it should not be called "Telegram for Linux" or use the official Telegram icon. Instead, it should have its own identity (e.g. "Paper Plane", with a custom icon). If it's made clear that it's a third party client it can be ok to mention the original service (e.g. "Tally for Plausible"), but in most cases it should be avoided.
```